src/fsharp/TypedTreeOps.fs

```diff
@@ -3717,7 +3717,7 @@ module DebugPrint =
         reprL
 
     and bindingL g (TBind(v, repr, _)) =
-        valAtBindL g v --- (wordL(tagText "=") ^^ exprL g repr)
```

Comment:
These are just cleanup to DebugPrint to give slightly better formatting when dumping expressions etc.